service.yaml

```diff
@@ -11,6 +11,7 @@ metrics: datadog
 releaseType: ""
 releaseOptions:
   allowPrereleases: true
+  prereleasesBranch: ""
```

Comment:
Should this have a value? 🤔

Reply:
Sorry about that, looks like the PR ran with RC versions. All fixed up now.